lib/sql.js

```diff
@@ -771,6 +771,7 @@ SQLConnector.prototype._buildWhere = function(model, where) {
       stmt.merge(columnName + ' IS NULL');
     } else if (expression && expression.constructor === Object) {
       var operator = Object.keys(expression)[0];
+      var options = expression.options;
```

Comment:
Have a look at the 3baf1242f28b1c8f9b30e0080f4560a3cca03fd7, is it what you meant?